### PR TITLE
Add docker server registry

### PR DESCRIPTION
**Pull request by Kartik494**
Fixes: #27937

### CODE REVIEW

File: content/en/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -201,7 +201,8 @@ to create a Docker registry Secret, you can do:
 kubectl create secret docker-registry secret-tiger-docker \
   --docker-username=tiger \
   --docker-password=pass113 \
-  --docker-email=tiger@acme.com
+  --docker-email=tiger@acme.com \
+  --docker-server=my-registry:5000
```
Comment (sftim) [inline content/en/docs/concepts/configuration/secret.md:205]: A Docker registry's server is a hostname, right?
```suggestion
  --docker-server=my-registry.example:5000
```

**Review by sftim — CHANGES_REQUESTED** [review]
You must also update the example JSON dump below the command so that it matches.

If you want to: create two examples, one simple and one fully developed. Use [tabs](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs) to let the reader pick which one they want to read.

File: content/en/docs/concepts/configuration/secret.md (hunk 1 above)
Comment (Kartik494) [reply content/en/docs/concepts/configuration/secret.md]: Added a commit for the same. For the JSON dump output i will add it accordingly asap
Thanks

**Comment by Kartik494** [comment]
Hi @sftim  if i understood it correctly i have to make two examples for users who want to use docker server registry and one for the user who don't want to use docker server registry?

**Comment by sftim** [comment]
You can either:
1. have just one example. The one example sets `--docker-server`, and the JSON output matches that.
2. have two examples. The first example doesn't set `--docker-server`, and the JSON output matches that. There is a second tab that does set `--docker-server` and for that tab the JSON output **does** show a custom registry URL.

Either is fine - pick which you prefer.

**Comment by Kartik494** [comment]
@sftim Ok, i would like to go with option 1 , i will add commit accordingly

**Comment by Kartik494** [comment]
@sftim added a commit for the same . PTAL

**Comment by Kartik494** [comment]
@sftim Could you please take a look on this.

Thanks

**Comment by Kartik494** [comment]
@sftim  Could you please take a look on this and let me know if any other  changes are needs to be done on this.
Thanks!

**Pull request merged**